### PR TITLE
Adds PR template for new workshops

### PR DESCRIPTION
**Pull request by vsmart**
Related to https://github.com/ClojureBridge/Workshops/pull/97

### CODE REVIEW

File: .github/post_template.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+layout:             post
+permalink:          :x: /events/2019-00-00-LOCATION
+country:            :x: The Netherlands
+city:               :x: Amsterdam
+street:             :x: Waag, Nieuwmarkt 4, 1012 CR Amsterdam
+latitude:           :x: 52.370411
+longitude:          :x: 4.90038
+start-date:         :x: 2019-04-11
+end-date:           :x: 2019-04-11
```
Comment (davidchambers) [inline .github/post_template.md]: Perhaps we should set these date strings to `1970-01-01` so they are more obviously placeholders.

File: .github/post_template.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+:x:
+[Waag](https://waag.org/), Nieuwmarkt 4, 1012 CR Amsterdam
+
+### Description
+
+:x:
+
+#### ClojureBridge
+
+:x:
```
Comment (davidchambers) [inline .github/post_template.md]: What should appear in the ClojureBridge section?

Comment (vsmart) [reply .github/post_template.md]: Hmm I just copied this from one of the events. We can remove these and add a note that organisers can add their own sub-headers if they'd like to 🤔

File: .github/post_template.md (hunk 1 above)
Comment (vsmart) [reply .github/post_template.md]: done

File: .github/post_template.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+start-date:         :x: 1970-01-01
+end-date:           :x: 1970-01-01
+registration-url:   :x: URL for registration
+city-image-url:     :x: assets/images/events/YOUR_EVENT_IMAGE
+gravatar-email:     :x: YOUR contact email
+organizers:         :x: [{"email": "YOUR_EMAIL", "github": "YOUR_GITHUB", "name": "YOUR_NAME", "twitter": "YOUR_TWITTER"}]
+sponsors:           []
+---
+
+<---- The above ⏫ sections is REQUIRED. Please fill out everything with an :x: ----->
```
Comment (davidchambers) [inline .github/post_template.md]:
```suggestion
<---- The above ⏫ section is REQUIRED. Please fill out everything with an :x: ----->
```

File: .github/post_template.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+street:             :x: Your Street
+latitude:           :x: Latitude (e.g. 52.370411)
+longitude:          :x: Longitude (e.g. 13.404954) 
+start-date:         :x: 1970-01-01
+end-date:           :x: 1970-01-01
+registration-url:   :x: URL for registration
+city-image-url:     :x: assets/images/events/YOUR_EVENT_IMAGE
+gravatar-email:     :x: YOUR contact email
+organizers:         :x: [{"email": "YOUR_EMAIL", "github": "YOUR_GITHUB", "name": "YOUR_NAME", "twitter": "YOUR_TWITTER"}]
+sponsors:           []
```
Comment (davidchambers) [inline .github/post_template.md]: I think we should include a placeholder sponsor so organizers can see the expected format.

**Comment by davidchambers** [comment]
Thank you, Franka!

**Pull request merged**